axelrod/result_set.py

```diff
@@ -45,6 +47,8 @@ def __init__(self, players, interactions, progress_bar=True):
             progress_bar : bool
                 Whether or not to create a progress bar which will be updated
         """
+        if game is not None:
+            self.game = game
```

Comment:
FYI, this is how things are currently done in the tournament class: https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tournament.py#L18.